### PR TITLE
feat: New command - create-dev-account

### LINKED ISSUE #106: `cargo near create-dev-account` command to ease development on testnet
NEAR CLI JS had `near dev-deploy` command that under the hood conditionally created a new account for users. I suggest avoiding this magic and instead, having a separate command to create a development account.

It is a going to be a shortcut to near-cli-rs command:

```
near account create-account sponsor-by-faucet-service <suggest-a-generated-name> ...
```

The full command I have in mind:

```
cargo near create-dev-account use-random-account-id autogenerate-new-keypair save-to-keychain network-config testnet create

cargo near create-dev-account use-specific-account-id frol.testnet autogenerate-new-keypair save-to-keychain network-config testnet create
```

Note: all the features of `near account create-account sponsor-by-faucet-service` should be available here as well, so near-cli-rs should be reused

### PR DESCRIPTION
**Pull request by FroVolod**
Resolves #106

### CODE REVIEW

File: cargo-near/src/commands/create_dev_account/mod.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+#[derive(Debug, EnumDiscriminants, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = near_cli_rs::GlobalContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+/// How do you cover the costs of account creation?
+pub enum CreateAccountMethod {
+    #[strum_discriminants(strum(
+        message = "use-random-account-id     - I would like to create a random account"
+    ))]
+    /// I would like to create a random account
```
Comment (frol) [inline cargo-near/src/commands/create_dev_account/mod.rs:20]:
```suggestion
        message = "use-random-account-id     - I would like to create a random account (useful for quick start development)"
    ))]
    /// Use a random name for account (useful for quick start development)
```

File: cargo-near/src/commands/abi_command/mod.rs [hunk 2]
```diff
@@ -79,18 +55,21 @@ impl interactive_clap::FromCli for AbiCommand {
...
     fn input_out_dir(
         _context: &near_cli_rs::GlobalContext,
     ) -> color_eyre::eyre::Result<Option<crate::types::utf8_path_buf::Utf8PathBufInner>> {
         Ok(None)
     }
 
+    #[allow(dead_code)]
     fn input_manifest_path(
         _context: &near_cli_rs::GlobalContext,
     ) -> color_eyre::eyre::Result<Option<crate::types::utf8_path_buf::Utf8PathBufInner>> {
```
Comment (frol) [inline cargo-near/src/commands/abi_command/mod.rs:75]: I believe these can be removed

File: cargo-near/src/commands/build_command/mod.rs [hunk 3]
```diff
@@ -89,18 +63,21 @@ impl interactive_clap::FromCli for BuildCommand {
...
     fn input_out_dir(
         _context: &near_cli_rs::GlobalContext,
     ) -> color_eyre::eyre::Result<Option<crate::types::utf8_path_buf::Utf8PathBufInner>> {
         Ok(None)
     }
 
+    #[allow(dead_code)]
     fn input_manifest_path(
         _context: &near_cli_rs::GlobalContext,
     ) -> color_eyre::eyre::Result<Option<crate::types::utf8_path_buf::Utf8PathBufInner>> {
```
Comment (frol) [inline cargo-near/src/commands/build_command/mod.rs:83]: Probably need to remove these as well

**Review by frol — CHANGES_REQUESTED** [review]

File: cargo-near/src/commands/abi_command/mod.rs (hunk 2 above)
Comment (FroVolod) [reply cargo-near/src/commands/abi_command/mod.rs:75]: Done

File: cargo-near/src/commands/build_command/mod.rs (hunk 3 above)
Comment (FroVolod) [reply cargo-near/src/commands/build_command/mod.rs]: Done

**Review by frol — APPROVED** [review]

**Pull request merged**